libs/ledger-live-common/src/families/cosmos/types.ts

```diff
@@ -135,21 +137,39 @@ export type NetworkInfoRaw = CosmosLikeNetworkInfoRaw & {
 export type CosmosOperation = Operation<CosmosOperationExtra>;
 export type CosmosOperationRaw = OperationRaw<CosmosOperationExtraRaw>;
 
-export type CosmosOperationExtra = {
+export type CosmosOperationExtra = OperationExtra & {
```

Comment:
If we already have a "OperationExtra" that is common to every operations, shouldn't this just be in the operation model itself ? (didn't really check what is in the type)

Reply:
It is an empty type. I added it to add type constraints to serialization functions.